docs/getting-started.md

```
@@ -37,12 +37,26 @@ npm link
+- `.diagram/brief.md` - human architecture brief
+- `.diagram/agent-context.json` - canonical AI-agent handoff
+- `.diagram/architecture.mmd` - first architecture diagram
+
+For PR review evidence, include refs:
+
+```bash
+archscope scan . --base origin/main --head HEAD
+```
+
+When refs resolve, the PR scan also writes
+`.diagram/pr-impact/pr-impact.json`. If the optional HTML report is not present,
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)